variables.tf

```diff
@@ -122,79 +122,84 @@ variable "master_authorized_networks_config" {
 
 variable "node_pools" {
```

Comment:
Should this be default node_pools

Reply:
both aks and gke went with node_pools and additional_node_pools.
not sure adding default makes a difference.